### PR TITLE
389 feedback screen visual design qa

### PR DESCRIPTION
**Pull request by megganeturner**
Addresses feedback in #389:
- remove sun icon & add new subtitle 'Let us know how you feel'
- centre the icons in the buttons, add labels, and add active state
- tweaks to the card such as border-radius, 'x' colour, `<p>` colour

I'm still waiting on a few translations - if I get them by EOD I'll update this branch, otherwise I can add them to my next branch (which looks at the feedback confirmation screen, so closely related)

Screen Shot 2021-02-11 at 3 02 59 pm
Screen Shot 2021-02-11 at 3 03 23 pm

### CODE REVIEW

File: src/riot/Lesson/LessonFeedback.riot.html [hunk 1]
```diff
@@ -1,32 +1,38 @@
...
                 </div>
                 <p>{ TRANSLATIONS.give_feedback() }</p>
             </div>
             <button class="btn-secondary" onclick="{clickedNoThanks}">{ TRANSLATIONS.no() }</button>
         </template>
         <template if="{state.madeFeedbackDecision}">
-            <div class="feedback-icon">
+            <!-- <div class="feedback-icon">
                 <div class="sun"></div>
                 <h2>{ TRANSLATIONS.feedback() }</h2>
```
Comment (katevoss) [inline src/riot/Lesson/LessonFeedback.riot.html:34]: The `TRANSLATIONS.feedback()` is no longer used in the modal so you can remove it from the `TRANSLATIONS` object in line 53.

File: src/scss/_lesson.scss [hunk 2]
```diff
@@ -461,32 +484,46 @@ LessonFeedback {
...
-            text-align: center;
-            border: none;
-            padding: 0;
+        & .button-label {
+            color: $gray-4;
+            margin-top: 11px;
         }
     }
 
+    // .feedback-icon {
```
Comment (katevoss) [inline src/scss/_lesson.scss:494]: You can remove the `.feedback-icon` class as it's not used anywhere else. Again I'm not sure why you commented it out but this whole section can be removed.

File: src/scss/_lesson.scss [hunk 3]
```diff
@@ -435,6 +438,26 @@ LessonFeedback {
                 outline: 0;
             }
 
+            &:active {
```
Comment (katevoss) [inline src/scss/_lesson.scss:441]: I don't think these styles appear, the smiley faces disappear as soon as you click one so there's no `:active` state.

**Review by katevoss — CHANGES_REQUESTED** [review]
Looks good, a few things to fix but should be fast.

File: src/scss/_lesson.scss (hunk 3 above)
Comment (megganeturner) [reply src/scss/_lesson.scss:441]: Oh true! I've just looked more closely at the design and can see the flow is actually `click an option (it highlights)` --> `click 'submit feedback' button` --> `confirmation screen`. I can work through this functionality today

**Comment by megganeturner** [comment]
Okay I've update the flow so that you can 'submit feedback' - I've kept the 'Maybe later' button in, as I feel that's a better user experience, but happy to remove it!

Screen Shot 2021-02-12 at 11 20 08 am
Screen Shot 2021-02-12 at 11 20 24 am

File: src/riot/Lesson/LessonFeedback.riot.html [hunk 4]
```diff
@@ -1,32 +1,37 @@
...
+                    <div class="button-container">
+                        <button onclick="{() => clickedThisLessonIs('great')}" class="thrilled {state.activeSelection === 'great' ? 'active' : ''}">
+                            <span class="thrilled-smiley"></span>
+                        </button>
+                        <p class="button-label">{ TRANSLATIONS.really_great() }</p>
+                    </div>
                 </div>
                 <p>{ TRANSLATIONS.give_feedback() }</p>
             </div>
+            <template if="{state.activeSelection}">
```
Comment (katevoss) [inline src/riot/Lesson/LessonFeedback.riot.html:29]: #### 1. When I click the button I see the `:active` style but when I finish "clicking", it reverts to the not `:active`. It should stay "active" once I have selected it.

#### 2. When this button appears, it shifts the content up (more noticable on wide and tall screens)
initial:


shifted:

**Review by katevoss — CHANGES_REQUESTED** [review]
One last issue with the buttons showing their colors. I'm good with having the "submit" and "Maybe Later" buttons, but let's make the "Submit" a primary button.

File: src/riot/Lesson/LessonFeedback.riot.html (hunk 4 above)
Comment (megganeturner) [reply src/riot/Lesson/LessonFeedback.riot.html]: Number 1 was caused by a smol merge issue - I'd changed `:active` to `.active`, but it had been overwritten. This is resolved now.

Number 2 has also been resolved, good pickup:
Screen Shot 2021-02-12 at 2 03 41 pm
Screen Shot 2021-02-12 at 2 03 49 pm

**Review by katevoss — APPROVED** [review]

**Pull request merged**